src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+  const heading = document.createElement('h2');
+  const body = document.createElement('p');
+
+  heading.textContent = title;
+  heading.classList = 'title';
```

Comment:
Avoid using 'classList =' to set classes as it overwrites any existing classes. Use 'classList.add()' instead.